infrastructure/idam-preview.tfvars

```diff
@@ -5,3 +5,5 @@ asp_name_override = "idam-idam-preview"
 asp_rg_override = "idam-idam-preview"
 
 capacity = 1
+
+strategic_policies_privateIpsFilterPattern = ""
```

Comment:
Could we experiment with setting this to
```
10.97\\.\\d+\\.\\d+
```

What I'm thinking is:
* this will filter out proxy IPs when accessing the app through the application gateway, hence allowing us to test manually without hitting the ForgeRock issue
* it will not filter out the app service IP, hence allowing the functional tests to pass 

This theory might prove wrong, but could we try it out please? :)